webodf/lib/ops/OdtDocument.js

```diff
@@ -444,15 +439,20 @@ ops.OdtDocument = function OdtDocument(odfCanvas) {
      * document's metadata, such as dc:creator,
      * meta:editing-cycles, and dc:creator.
      * @param {!ops.Operation} op
+     * @param {!Array.<!ops.Operation.Event>} events
+     * @return {undefined}
      */
-    function handleOperationExecuted(op) {
+    function finishOperationExecution(op, events) {
```

Comment:
I like this approach *much* better than updating the document from within the event!